With WoodStox 7.0.0 and Metro / WebServices 4.0.4 and  jakarta.xml.ws-api 4.0.2:

```
[ERROR] Failures: 
[ERROR]   ClusterITest.checkClusterTest:135->getURL:239
[ERROR]   ClusterITest.checkDeploymentTest:147->getURL:239
[ERROR]   ClusterITest.deleteClusterTest:218 
Expected: asadmin succeeded
     but: was <Command delete-cluster failed.
remote failure: Configuration com.sun.enterprise.config.serverbeans.Cluster not deleted. Cluster eec1 contains server instances eein1-with-a-very-very-very-long-name,eein2, and must not contain any instances

>
[ERROR]   ApplicationITest.listSubComponents:125->RestTestBase.undeployApp:208 
Expected: <200>
     but: was <404>
[ERROR]   ApplicationITest.testApplicationDeployment:67->RestTestBase.undeployApp:208 
Expected: <200>
     but: was <404>
[ERROR]   ApplicationITest.testApplicationDeploymentWithDefaultContextRoot:80->RestTestBase.undeployApp:208 
Expected: <200>
     but: was <404>
[ERROR]   ApplicationITest.testApplicationDisableEnable:86->RestTestBase.deployApp:190 
Expected: <200>
     but: was <400>
[ERROR]   ApplicationITest.testCreatingAndDeletingApplicationRefs:155->RestTestBase.undeployApp:208 
Expected: <200>
     but: was <404>
[ERROR]   ApplicationITest.testGetContextRoot:172->RestTestBase.undeployApp:208 
Expected: <200>
     but: was <404>
[ERROR]   ApplicationITest.testUndeploySubActionWarnings:188->RestTestBase.deployApp:190 
Expected: <200>
     but: was <400>
[ERROR]   ElementStarITest.testApplications:73->RestTestBase.deployApp:190 
Expected: <200>
     but: was <400>
[INFO] 
[ERROR] Tests run: 148, Failures: 11, Errors: 0, Skipped: 5
```

Seems we still have some way to go. @dmatej any idea?

--- a/appserver/pom.xml
+++ b/appserver/pom.xml
@@ -138,7 +138,7 @@
 
         <!-- Used for Jakarta SOAP (XML Web Services) -->
         <xmlsec.version>4.0.2</xmlsec.version>
-        <woodstox.version>6.6.2</woodstox.version>
+        <woodstox.version>7.0.0</woodstox.version>
         <stax2-api.version>4.2.2</stax2-api.version>
 
         <!-- Jakarta CDI -->

--- a/nucleus/parent/pom.xml
+++ b/nucleus/parent/pom.xml
@@ -88,7 +88,7 @@
         <!-- Jakarta SOAP (with Attachments)
        <jakarta.xml.soap-api.version>3.0.0-RC2</jakarta.xml.soap-api.version>
         -->
-        <webservices.version>4.0.3</webservices.version>
+        <webservices.version>4.0.4</webservices.version>
 
         <!-- Jakarta Inject -->
         <jakarta.inject-api.version>2.0.1.MR</jakarta.inject-api.version>

